joy_song/about2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang = "en-US">
+    <head>
+        <!--Ideas:
+            1) Add dropdown menu when hovering over "Blog", maybe stages of life/types of content?
+            2) Potentially change up color scheme (still looks a bit extra)
+        -->
+        <title>About Me</title>
+        <link rel="stylesheet" href="style2.css">
+    </head>
+    <body>
+        <div class="topnav">
+            <a href="home.html">Home</a>
+            <a class="active" href="about2.html">About</a>
+            <a href="blog.html">Blog</a>
+            <a href="projects.html">Projects</a>
+        </div>
+        <div class="header">
+            <h1> 
+                About me
+            </h1>
+        </div>
+        <div id="container">
```

Comment:
This is OK here when your project is not very big, but make sure to make your classes and IDs more specific to that chunk of HTML. I know my examples also just have "container" but important to note for the future.